libraries/eosiolib/contract.hpp

```diff
@@ -61,6 +67,13 @@ class contract {
        */
       name _self;
 
+      /**
+       * The code name of this contract
```

Comment:
I would instead describe this as something like the following:

`The code name of the action this contract is processing.`